src/popup/popup.html

```diff
@@ -33,7 +33,7 @@
           <input type="text" placeholder="search the commons..." id="section-search-input" autofocus />
           <i class="fa fa-search" id="search-icon"></i>
           <i class="fa fa-filter" id="filter-icon"></i>
-          <button class="clear-search-button">X</button>
+          <i class="fa fa-times-circle clear-search-button" style="color: #ff4e14; right: 12%"></i>
```

Comment:
don't add inline css. Define it in `_search.scss`

Reply:
I first tried that, but it is getting overwritten. Can I use !important?

Reply:
Okay. It will be fixed soon.